pkg/aws/eni/limits/limits.go

```diff
@@ -121,6 +121,15 @@ func populateStaticENILimits() {
 		"c6gn.large":    {Adapters: 3, IPv4: 10, IPv6: 10},
 		"c6gn.medium":   {Adapters: 2, IPv4: 4, IPv6: 4},
 		"c6gn.xlarge":   {Adapters: 4, IPv4: 15, IPv6: 15},
+		"c7g.12xlarge":  {Adapters: 8, IPv4: 30, IPv6: 30},
```

Comment:
out of curiousity, how do get details for this new instance types ? I am checking lastest CLI command, but c7g is not available in output. Thanks.

https://docs.aws.amazon.com/cli/latest/reference/ec2/describe-instance-types.html

Reply:
(sorry about multiple comments with same content, seems like github was having a short issue)

Reply:
I didn't notice your email, so all good :100: